content/04.results.md

```diff
@@ -1,7 +1,42 @@
-# Results
+## Results
 
-## Random Sampling of Generative Model
-place the grid aurocs here for generative model
+### Generative Model Using Randomly Sampled Label Functions
+
+To evaluate the feasibility of label function reuse we used a random sampling method.
+Our baseline model consisted of training the generative model with only distant supervision label functions for each edge type.
+We report the results in the form of area under the precision recall curve (AUPR) (Figure {@fig:gen_model_auprc}) and area under the receiver operating curve (AUROC) (Figure {@fig:gen_model_auroc}).  
+The general trend is performance increases as the number of edge specific label functions are added on top of the edge specific baseline models.
+This trend is shown on the on-diagonals of figure {@fig:gen_model_auprc}) and figure {@fig:gen_model_auprc}.
+A quintessential example of this trend is with the compound treats disease (CtD) edge type.
+The baseline model starts off with an auroc score of 52% and an auprc of 28%, then increases to 76% and 49% as the number of edge specific label functions are incorporated. 
+The disease associates gene (DaG) edge has a similar trend with performance starting off with a auroc score of 56% and auprc score of 41% then increasing to 62% and 45% respectively.
+Both the Compound binds Gene (CbG) and Gene interacts Gene (GiG) edges have an increasing trend (auroc of 78% to 82% and 66% to 77% respectively), but plateaus as the number of label functions increases.  
+For the off-diagonals performance varies upon edge type as some edges can aide in predicting others.
+Looking at the DaG row, performance increases regardless of the edge type source.
+This trend changes for CtD as performance starts to increase, when only adding one label function, then plateaus (auroc) or decreases (auprc) when more label functions are added.
+For CbG performance decreases when using label functions from the DaG and CtD category.
+As expected performance increases for both auprc and auroc when using label functions from the GiG edge type.
+This trend is shared when predicting the GiG edge; however the magnitude of performance is low.  
+Lastly, performance increases (auroc and auprc) for both DaG and CtD, when sampling from all label functions at once.
+Interestingly, CbG and GiG share a similar trend of increasing in performance when one random label function is used, but drastically decreases as more label functions are incorporated.
+
+![
```

Comment:
You may want to move this figure to earlier in the section so that there's some up front figures for folks to look at.